app/views/thesis/new.html.erb

```diff
@@ -124,6 +124,7 @@
     </div>
 
     <%= f.association :copyright, as: :select,
+                                  selected: 1,
```

Comment:
@jazairi It looks like we set the ID is consistent in the PR builds as well so while not ideal, it's probably a risk we can accept as there is little reason we'd be changing the Author copyright to be something inherently different under the good.

That said, I believe this ID is wrong as Author is ID 2 (in the PR build and prod at least).

Reply:
@JPrevost Thanks for pointing that out! While fixing that, I noticed that I'd forgotten to include license validation/required field indicator in the edit view, so I added that, too.